chef/data_bags/crowbar/bc-template-keystone.json

```diff
@@ -31,6 +31,7 @@
         "api_host": "0.0.0.0",
         "admin_port": 35357,
         "admin_host": "0.0.0.0",
+        "version": "v2.0",
```

Comment:
Can we add a validation check in the model to make sure that no crazy string is used here?